woocommerce/woocommerce-ios
# Login: Remove the survey at the end of the login onboarding flow

Closes: #8056 


### Description

After launching the survey in the last quarter, we've collected enough data to understand users' intentions in using the app - we decided to remove the survey as it's a friction for the login onboarding flow (internal discussion: p1667902271165339-slack-C03L1NF1EA3). 

This PR removes all of the implementation related to the onboarding survey.

### Testing instructions

- Delete and reinstall the app.
- On the onboarding carousel, tap "Next" until you reach the end of the carousel.
- Continue tapping "Next" - notice that the carousel is then dismissed and you are navigated to the login prologue screen. The survey should no longer be available.

### Screenshots


https://user-images.githubusercontent.com/5533851/200743736-6f1ff5db-70be-4a39-bec7-c2b363df3840.mp4



---
